Assets/Features/Autopilot/Provider Experimental/AutopilotDebug.cs

```diff
@@ -0,0 +1,97 @@
+﻿using EdyCommonTools;
+using Perrinn424.AutopilotSystem;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AutopilotDebug : MonoBehaviour
+{
+    public Rigidbody rb;
+    public RecordedLap lap;
+
+    private NearestSegmentSearcher segmentSearcher;
+    private Path path;
+    private PidController pid;
```

Comment:
Is this pid controller file that is written, and you are just calling it?  We didnt rewrite this right?

Reply:
The PID controller is in the Edy's library. We can write our own but I think this works pretty well